service/history/ndc/workflow_resetter_test.go

```diff
@@ -283,22 +281,15 @@ func (s *workflowResetterSuite) TestReplayResetWorkflow() {
 	baseBranchToken := []byte("some random base branch token")
 	baseRebuildLastEventID := int64(1233)
 	baseRebuildLastEventVersion := int64(12)
-	baseNodeID := baseRebuildLastEventID + 1
 
 	resetBranchToken := []byte("some random reset branch token")
 	resetRequestID := uuid.New()
 	resetHistorySize := int64(4411)
 	resetMutableState := workflow.NewMockMutableState(s.controller)
 
-	shardID := s.mockShard.GetShardID()
-	s.mockExecutionMgr.EXPECT().ForkHistoryBranch(gomock.Any(), &persistence.ForkHistoryBranchRequest{
```

Comment:
It is useful to see if the argument for ForkHistoryBranch() is what we expect. Could you change only the field in the request that was different? Sorry If I missed this in last review.

Reply:
As we discussed over slack, I agree with your reasoning that ForkHistoryBranch() is not called directly by replayResetWorkflow() and this check may be unnecessary.